An_attempt_at_Cosine_similarity.py

```diff
@@ -0,0 +1,221 @@
+    input_file = open('nietzsche.pickle','rb')
+    reloaded_copy_of_texts3 = pickle.load(input_file)
+
+    ethics = requests.get('http://www.gutenberg.org/cache/epub/8438/pg8438.txt').text
+    #print(oliver_twist_full_text)
+
+    f = open('ethics.pickle','wb')
+    pickle.dump(ethics,f)
+    f.close()
+
+    # Load data from a file (will be part of your data processing script)
+    input_file = open('ethics.pickle','rb')
+    reloaded_copy_of_texts4 = pickle.load(input_file)
+
+    utilitarianism = requests.get('http://www.gutenberg.org/cache/epub/11224/pg11224.txt').text
+    #print(oliver_twist_full_text)
+
+    f = open('utilitarianism.pickle','wb')
+    pickle.dump(utilitarianism,f)
+    f.close()
+
+    # Load data from a file (will be part of your data processing script)
+    input_file = open('utilitarianism.pickle','rb')
+    reloaded_copy_of_texts = pickle.load(input_file)
+
+
+"""
+html = BeautifulSoup(requests.get("http://www.worldfuturefund.org/Reports2013/hitlerenablingact.htm").text, 'lxml')
+html.find('p')
+str(html.find('p'))
+
+html1 = BeautifulSoup(requests.get("https://www.marxists.org/reference/archive/stalin/works/1941/11/07.htm").text, 'lxml')
+html1.find('p')
+str(html1.find('p'))
+
+html2 = BeautifulSoup(requests.get("http://www.politico.com/story/2016/07/full-transcript-donald-trump-nomination-acceptance-speech-at-rnc-225974").text, 'lxml')
+html2.find('p')
+str(html2.find('p'))
+
+html3 = BeautifulSoup(requests.get("http://obamaspeeches.com/").text, 'lxml')
+html3.find('p')
+str(html3.find('p'))
+
+url = "http://www.worldfuturefund.org/Reports2013/hitlerenablingact.htm"
+content = urllib2.urlopen(url)
+
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+analyzer = SentimentIntensityAnalyzer()
+analyzer.polarity_scores('Software Design is my favorite class!')
+
+"""
+def process_file(name_of_file):
+    hist = dict()
+    open_file = open(name_of_file, 'rb')
+    for l in open_file:
+        process_line(l,hist)
+    return hist
+
+def process_line(line,hist):
+    # line = line.replace('-', ' ')
+
+    for word in line.split():
+        # word = word.strip(string.punctuation + string.whitespace) #TODO y no work
+        word = word.lower()
+        hist[word]=hist.get(word,0)+1
+
+constitution_hist = process_file('constitution.pickle')
+ethics_hist = process_file('ethics.pickle')
+communist_manifesto_hist = process_file('communist_manifesto.pickle')
+nietzsche_hist = process_file('nietzsche.pickle')
+utilitarianism_hist = process_file('utilitarianism.pickle')
+
+#the inverse function takes a dictionary and returns a list of tuples that are sorted by frequency and word.
+"""def inverse(dictionary):
+    inverse_list = []
+    for key in (dictionary):
+        if dictionary[key] > 20:
+            t=(dictionary[key]/(len(dictionary)),key)
+            inverse_list.append(t)
+    new_list = sorted(inverse_list)
+    return new_list"""
+
```

Comment:
For a final submission of your code, you want to delete all unnecessary parts of the code. Commented lines of code that you used for debugging or not a part of your final code should be deleted.